data.sql

```diff
@@ -1,5 +1,5 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+insert into animals(name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Agumon', '2020-02-03', 0, true, 10.23);
+insert into animals(name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Gabumon', '2018-11-15', 2, true, 8), ('Pikachu', '2021-01-07', 1, false, 15.04); 
+insert into animals(name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Devimon', '2017-05-12', 5, true, 11); 
```

Comment:
Great job @Bateyjosue 🤝

- [ ] You may add all the screenshots to your Pull requests as stated in the project requirements.
- [ ] Kindly maintain all `weight_kg` values for the animals table as `boolean` values.